Catappult/appcoins-wallet-android
# [feature/APPC-2571] Adyen exceptions feedback

**What does this PR do?**

   Implements 5 new Adyen payment error messages in the Adyen payment flow and Top up flow

**Database changed?**

   No

**Where should the reviewer start?**

- AdyenResponseMapper.kt 
- AdyenPaymentFragment.kt
- AdyenTopUpFragment.kt

**How should this be manually tested?**

  We only found a credit card that could give one of the 5 errors, but the implementation should cover the other the same way.
  To test use 
  number: 6033 4225 5384 5003 
  date: 01/2023
  cvc: 418
  - Make a payment with trivial drive
  - Make a top up payment

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/APPC-2571


**Code Review Checklist**
